pop_tools/grid.py

```diff
@@ -320,6 +361,16 @@ def get_grid(grid_name, scrip=False):
     return dso
 
 
+@jit(nopython=True, parallel=True)
+def _generate_KMU(KMT):
+    """Computes KMU from KMT."""
+    KMU = np.zeros_like(KMT)
```

Comment:
How about initializing `KMU` outside of the function so as to avoid having to initialize it for every function call? This change would involve passing `KMU` as an input, and updating it inside the function. We could then change the function signature to 

```python
def _generate_KMU(KMT, KMU)
```

Reply:
This will only be called once upon calling `get_grid()` so I figure it's fine to initialize in there. I went ahead and switched it though. I'm returning the modified `KMU` since that's generally how I write this kind of thing.

But per your pass-by-reference model, do I need to return it at all? Or will it automatically be modified? I'm still blown away by this. I thought modifications inside functions were just to the private variable, not the global variable.